openpype/hosts/houdini/api/plugin.py

```diff
@@ -256,8 +256,14 @@ def update_instances(self, update_list):
                 new_values,
                 update=True
             )
+
+            # Pop these keys as they error with setParms
+            new_values.pop("creator_attributes", None)
+            new_values.pop("publish_attributes", None)
+
             # Update values
-            instance_node.setParms(new_values)
+            if new_values:
+                instance_node.setParms(new_values)
```

Comment:
No no no no no no.

⚠️ This is NOT the way.

`self.imprint` should be taking care or setting the new values - why do we need to explicitly call `setParms`?

What is this a workaround for?

Reply:
Ah, I see `instance_node.setParms()` is already in `develop`. That should **NOT** be there.

Reply:
I'm sorry about that. Let's fix it. 

> `self.imprint` should be taking care or setting the new values, I'm afraid it doesn't happen! 

I tested it a lot and it didn't work until I explicitly called `setParms` 
Maby I'm missing something. I don't know.
https://github.com/ynput/OpenPype/pull/5621#discussion_r1355611105

Reply:
The bug then is in `imprint` implementation. I have no time to look into it now - but it's up to the `imprint` call to decide what to store where, and update the values.